raiden/blockchain/events.py

```diff
@@ -57,12 +63,16 @@ def get_contract_events(
     `contract_address` that match the filters `topics`, `from_block`, and
     `to_block`.
     """
-    events = chain.client.get_filter_events(
-        contract_address,
-        topics=topics,
-        from_block=from_block,
-        to_block=to_block
-    )
+    try:
```

Comment:
I think we move this into the client, otherwise we have to repeat it everywhere the `get_filter_events`  method is used.

Reply:
ah, you are right, that makes much more sense.